rswag-specs/lib/rswag/specs/swagger_formatter.rb

```diff
@@ -58,6 +58,9 @@ def stop(_notification = nil)
                 if is_hash && value[:parameters]
                   schema_param = value[:parameters]&.find { |p| (p[:in] == :body || p[:in] == :formData) && p[:schema] }
                   mime_list = value[:consumes] || doc[:consumes]
+
+                  value[:parameters].each {|p| p.delete(:getter) } # remove service field
```

Comment:
@pechorin, I'm wondering — can the `:getter` field be removed in the [`remove_invalid_operation_keys!`](https://github.com/rrebellion/rswag/blob/20fc48b088dbc6b2c35cdd1ee1a1903efd41dc63/rswag-specs/lib/rswag/specs/swagger_formatter.rb#L212) method?